FBC/ProductsAndServices/ClientsAndAccounts.rdf

```diff
@@ -106,26 +110,50 @@
 				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-pas-caa;hasAccountCloseDate"/>
+				<owl:onClass rdf:resource="&fibo-fbc-pas-caa;hasAccountCloseDate"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
 				<owl:onClass rdf:resource="&fibo-fnd-arr-doc;Record"/>
 				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-pas-caa;hasAccountOpenDate"/>
+				<owl:onClass rdf:resource="&fibo-fbc-pas-caa;hasAccountOpenDate"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-pas-caa;hasBalance"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-pas-caa;Balance"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-pty-pty;hasPartyInRole"/>
 				<owl:someValuesFrom rdf:resource="&fibo-fbc-pas-caa;AccountProvider"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>account</rdfs:label>
-		<skos:definition>a container for records associated with a business arrangement for regular dealings or services (such as personal or professional services, banking)</skos:definition>
-		<fibo-fnd-utl-av:adaptedFrom>Barron&apos;s Dictionary of Business and Economics Terms, Fifth Edition, 2012</fibo-fnd-utl-av:adaptedFrom>
-		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.merriam-webster.com/dictionary/account</fibo-fnd-utl-av:adaptedFrom>
+		<skos:definition>container for records associated with a business arrangement for regular dealings or services (such as personal or professional services, banking)</skos:definition>
 		<fibo-fnd-utl-av:explanatoryNote>In general, an account is associated with a contractual relationship between a buyer and seller under which payment may be made at a later time.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fbc-pas-caa;AccountCloseDate">
```

Comment:
I don't know why AccountCloseDate needs an explicit class - why not a property of type ExplicitDate?

Reply:
AccountCloseDate and AccountOpenDate are important dates for most banks, and typically called reified in their KGs.  But, for FIBO I have now changed this to ExplicitDate, and removed the date classes, which banks can add as extensions.  Since they all use this pattern I thought I would preempt the request, but it's ok as ExplicitDate for the time being.